doc/user/content/transform-data/patterns/percentiles.md

```diff
@@ -144,9 +136,8 @@ their relative error becomes less relevant (e.g. 10000 ms and 10001 ms are
 almost the same and can reside in the same bucket).
 
 In the example below, to reduce the number of buckets, the values are first
-decomposed into their floating point representation (i.e., `mantissa *
-2^exponent`), and then with the precision of the mantissa lowered,
-reconstructed for the respective bucket value.
+decomposed into `signed_mantissa * 2^exponent`, and then with the precision of
```

Comment:
`signed_mantissa` eventually becomes `mantissa --signed`. Perhaps we want to bring those together somehow. It seems fine to put here "`mantissa * 2^exponent` where the mantissa is signed"? I think it's perhaps a terminological bug in that the thing we are discussing is the "[significand](https://en.wikipedia.org/wiki/Significand)", and the mantissa is only the part after the decimal point in the normalized significand.
<img width="579" alt="Screenshot 2025-03-27 at 12 38 02 PM" src="https://github.com/user-attachments/assets/370ebb9d-43a7-43fa-9c8e-fc5b2e11b1d2" />